src/test/java/org/gridsuite/loadflow/server/LoadFlowControllerTest.java

```diff
@@ -664,53 +695,6 @@ public void getSpecificParametersTest() throws Exception {
 
     }
 
-    @Test
-    public void geLimitTypesTest() throws Exception {
-        MvcResult mvcResult = mockMvc.perform(get("/" + VERSION + "/limit-types"))
-                .andExpectAll(
-                        status().isOk(),
-                        content().contentType(MediaType.APPLICATION_JSON)
-                ).andReturn();
-
-        String resultAsString = mvcResult.getResponse().getContentAsString();
-        List<LimitViolationType> limitTypes = mapper.readValue(resultAsString, new TypeReference<>() {
-        });
-        assertEquals(2, limitTypes.size());
-        assertTrue(limitTypes.contains(LimitViolationType.HIGH_VOLTAGE));
-        assertTrue(limitTypes.contains(LimitViolationType.LOW_VOLTAGE));
-        assertFalse(limitTypes.contains(LimitViolationType.CURRENT));
```

Comment:
Why did you remove the tests on the other limit types ? Given that you filter on what is in the results it might be useful to have some limit types not contained in the resultats and tested.